systemtest/src/test/java/io/strimzi/systemtest/mirrormaker/MirrorMakerST.java

```diff
@@ -737,16 +737,9 @@ void testHostAliases() {
 
         String mmPodName = kubeClient().listPods(Labels.STRIMZI_KIND_LABEL, KafkaMirrorMaker.RESOURCE_KIND).get(0).getMetadata().getName();
 
-        LOGGER.info("Trying to ping strimzi.io by ping strimzi command");
-        String output = cmdKubeClient().execInPod(mmPodName, "ping", "-c", "5", "strimzi").out();
-
-        LOGGER.info("Checking output of ping");
-        assertThat(output, Matchers.containsString("PING strimzi (34.89.152.196)"));
-        assertThat(output, Matchers.containsString("5 packets transmitted, 5 received"));
-
         LOGGER.info("Checking the /etc/hosts file");
-        output = cmdKubeClient().execInPod(mmPodName, "cat", "/etc/hosts").out();
-        assertThat(output, Matchers.containsString("# Entries added by HostAliases.\n34.89.152.196\tstrimzi"));
+        String output = cmdKubeClient().execInPod(mmPodName, "cat", "/etc/hosts").out();
+        assertThat(output, Matchers.containsString(etcHostsData));
```

Comment:
Sorry that I'm reviewing this, but can you please remove the `Matchers`? It's leftover from my PR...